src/js/client/handlebars.js

```diff
@@ -0,0 +1,179 @@
+// A client-side module that provides various template handling capabilities, and which wires in any
+// child components with the grade `gpii.templates.hb.helper` as handlebars helpers.
+//
+// Requires Handlebars.js and Pagedown (for markdown rendering)
+
+/* global fluid, jQuery, Handlebars */
+(function ($) {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+    fluid.registerNamespace("gpii.templates.hb.client");
+
+    gpii.templates.hb.client.addHelper = function (that, component) {
+        var key = component.getHelperName();
```

Comment:
I refactored to access `options.helperName` instead of the get wrapper.  To keep the same level of protection, I made the base grade check for the option during the `onCreate` event and fail if it's not found.